### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by s-santillan**
Added config auto to code sample
Updated Playground descriptor as "tool"

PR checklist:
- [x] Documentation is up-to-date
- [x] Changelog is up-to-date
- [x] Change has no security implications (otherwise, ping security team)

### CODE REVIEW

**Review by ievans — CHANGES_REQUESTED** [review]
Can we keep the language consistent about how it uses the git URL to identify your project so users aren't surprised?

File: README.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ The Semgrep ecosystem includes:
 
 - Semgrep - the open-source command line tool at the heart of everything (this project)
 - [Semgrep CI](https://semgrep.dev/docs/semgrep-ci/) - a specialized Docker image for running Semgrep in CI environments
-- [Semgrep Playground](https://semgrep.dev/editor) - an online interactive editor for writing and sharing rules
+- [Semgrep Playground](https://semgrep.dev/editor) - an online interactive tool for writing and sharing rules
```
Comment (ievans) [inline README.md]: Why this change? Editor is more descriptive than tool, even if it's no longer the primary name

Comment (s-santillan) [inline README.md]:
```suggestion
- [Semgrep Playground](https://semgrep.dev/editor) - an online interactive rule builder for writing and sharing rules
```

File: README.md [hunk 2]
```diff
@@ -89,6 +89,10 @@ Once installed, Semgrep can run with single rules or entire rulesets. Visit [Doc
 # Check for Python == where the left and right hand sides are the same (often a bug)
 $ semgrep -e '$X == $X' --lang=py path/to/src
 
+# Fetch rules automatically by examining your code's makeup (your source code
```
Comment (s-santillan) [inline README.md]:
```suggestion
# Fetch rules automatically by setting the `--config auto` flag. 
# This will fetch rules relevant to your project from Semgrep Registry.
# The name of your project will be sent to Semgrep Registry as an identifier 
# to make selecting relevant rules fast next time; source code will not be uploaded.
```

File: README.md [hunk 3]
```diff
@@ -89,6 +89,13 @@ Once installed, Semgrep can run with single rules or entire rulesets. Visit [Doc
 # Check for Python == where the left and right hand sides are the same (often a bug)
 $ semgrep -e '$X == $X' --lang=py path/to/src
 
+# Fetch rules automatically by setting the `--config auto` flag. 
+# This will fetch rules relevant to your project from Semgrep Registry.
+# The name of your project will be sent to Semgrep Registry as an identifier 
+# to make selecting relevant rules fast next time; source code will not be uploaded.
```
Comment (s-santillan) [inline README.md]:
```suggestion
# to make selecting relevant rules fast next time;
# source code will not be uploaded.
```

File: README.md [hunk 4]
```diff
@@ -89,6 +89,13 @@ Once installed, Semgrep can run with single rules or entire rulesets. Visit [Doc
...
+# is not uploaded anywhere)
```
Comment (s-santillan) [inline README.md]:
```suggestion
```

**Comment by s-santillan** [comment]
> Can we keep the language consistent about how it uses the git URL to identify your project so users aren't surprised?

Did this in the last few edits. I kept the paragraph text in the sample code to keep the flow of this repo's readme (whereas in semgrep-rules it was a paragraph).

**Review by ievans — APPROVED** [review]
